pkgs/tools/package-management/nix-folder2channel/default.nix

```
@@ -0,0 +1,34 @@
+{
+  stdenv, writeScriptBin, buildEnv,
+  git,
+}:
```

```suggestion
{ stdenv, writeScriptBin, buildEnv, git }:
```

You are using a strange formatter. Give https://github.com/nix-community/nixpkgs-fmt a try.

---

Hi @SuperSandro2000, thanks for review. 
I have tried nixpkgs-fmt(1.0.0), but it always gives me a strange format like this.
```
 { stdenv
 , writeScriptBin
 , buildEnv
 , git
 ,
 }:
```
May you teach me how to set up `nixpkgs-fmt` to get a compact format.  Thanks.
This is the command I used `nixpkgs-fmt default.nix`

---

The nixpkgs-fmt only handle that all the dependency at one line, I will take care about this, thanks.

---

It can only do
```
{ stdenv
 , writeScriptBin
 , buildEnv
 , git
 ,
 }:
```
or
```
{ stdenv, writeScriptBin, buildEnv, git }:
```